pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+Predicting the value of  macroscopic pKas based between 2 and 12.
+- Fill one `typeIII_macroscopic_pKas.csv` template file for all predicted molecules.
+- Report as many macroscopic pKas based on your predictions.
+- Report pKa values with 2 decimals (e.g. 10.71).
+- Reporting standard error of the mean (SEM) is optional and should be reproted with 2 decimals. (e.g. 1.02).
+
+## A warning about enumerated microstates and requesting missing microstates
+A list of microstates and microstate IDs were generated for each molecule to aid parsing of submissions. Enumerated list of microstates was not created with the intend to guide computational predictions. It is possible that some relevant microscopic species are missing from these lists. If your predicted structure is not already included in the microstates list, contact Mehtap Isik (mehtap.isik@choderalab.org). Please send us 2D structure depiction and canonical isomeric SMILES of your proposed microstate. We will assign a unique microstate ID and include it in the analysis.  
+
+Please do not create a microstate ID yourself. It is important that challenge organizers assign unique microstate IDs and keep track.  
+
+## Submission of multiple predictions
+
+Some participants use SAMPL to help evaluate various computational methods. To accommodate this, multiple prediction sets from a single research group or company are allowed, even for same type of predictions if they are made by different methods.
+
+## Uploading your predictions
+
+D3R is currently outfitting the SAMPL6 page with the ability to accept your uploaded predictions. As soon as this is ready, you may upload your predictions. If you want to upload more than one type predictions (see description of three pKa prediction types above) or different set of predictions (same type but generated by different methods), each must be uploaded as a separate file. Please use the template provided, as the predictions will be parsed and analyzed with automated scripts. Please include predictions related all molecules with same method and same submission type in one file. 
+
+We encourage submitting predictions in all three formats and for all 24 molecules when possible. Incomplete submissions - such as for a subset of compounds - will also be accepted, but will not necessarily be evaluated together with the rest of the submissions. However, we would emphasize that omission of SEM estimates will not cause a submission to be regarded as incomplete.
+
+Names of the prediction files must begin with the name of the submission type (i.e., typeIII), and your name and must end with an integer indicating set of prediction. For example, if you want to submit one prediction file for type III prediction (macroscopic pKas), you might name it `typeIII-myname-1.csv`, where myname is arbitrary text of your choice. If you submit two prediction files of the same submission type, you might name them `typeIII-myname-1.txt` and `typeIII-myname-2.txt`.
```

Comment:
Replace might with would as you are expecting them to follow your directions.

Reply:
Great! Corrected this. Thanks.